crates/ruff/src/message/grouped.rs

```diff
@@ -116,47 +113,10 @@ impl Display for DisplayGroupedMessage<'_> {
             },
         )?;
 
-        if let Some(source) = &message.source {
-            let suggestion = message.kind.suggestion.clone();
-            let footer = if suggestion.is_some() {
-                vec![Annotation {
-                    id: None,
-                    label: suggestion.as_deref(),
-                    annotation_type: AnnotationType::Help,
-                }]
-            } else {
-                vec![]
-            };
-            let label = message.kind.rule().noqa_code().to_string();
-            let snippet = Snippet {
-                title: None,
-                footer,
-                slices: vec![Slice {
-                    source: &source.contents,
-                    line_start: message.location.row(),
-                    annotations: vec![SourceAnnotation {
-                        label: &label,
-                        annotation_type: AnnotationType::Error,
-                        range: source.range,
-                    }],
-                    // The origin (file name, line number, and column number) is already encoded
-                    // in the `label`.
-                    origin: None,
-                    fold: false,
-                }],
-                opt: FormatOptions {
-                    #[cfg(test)]
-                    color: false,
-                    #[cfg(not(test))]
-                    color: colored::control::SHOULD_COLORIZE.should_colorize(),
-                    ..FormatOptions::default()
-                },
-            };
-            // Skip the first line, since we format the `label` ourselves.
-            let message = DisplayList::from(snippet);
+        {
+            use std::fmt::Write;
```

Comment:
Imported here for shadowing, to avoid the need to alias?

Reply:
Yeah. The alternative is to use `use std::fmt::{Write as WriteFmt}` at the top. I guess it was just easier to type it here than jumping to the top and manually at the use